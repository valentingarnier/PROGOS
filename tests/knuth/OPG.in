3 K
0 0
1 1
2 0
3 0
