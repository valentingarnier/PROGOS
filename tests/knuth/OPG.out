What size (2, 3, 4, 5)? What strategy ([B]rute force, B[i]tfield, [K]nuth)? Y Y B
Please score attempt (positions, colors): P R G
Please score attempt (positions, colors): O R G
Please score attempt (positions, colors): O P G
Please score attempt (positions, colors): Found solution: O P G
